templates/ci/github-actions/php.yml

```
@@ -0,0 +1,53 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  push:
```

YAMLlint (1.37.1)

[error] duplication of key "push" in mapping

(key-duplicates)